src/goDebugConfiguration.ts

```diff
@@ -54,6 +54,15 @@ export class GoDebugConfigurationProvider implements vscode.DebugConfigurationPr
 			debugConfiguration['dlvLoadConfig'] = dlvConfig['dlvLoadConfig'];
 		}
 
+		let debugMode;
+		let testFileRegExp = RegExp('.*_test\.go$', 'g');
+		const currentFileName = vscode.window.activeTextEditor.document.fileName;
+		if (testFileRegExp.test(currentFileName)) {

```

Comment:
Isnt it simpler with `if (activeEditor.document.fileName.endsWith('_test.go'))` ?
I have a tendency to avoid regex when possible.